gen/definitions/bgp_advertised_address.yaml

```diff
@@ -0,0 +1,92 @@
+---
+name: BGP Advertised Prefix
+class_name: bgpAdvPrefix
+dn: sys/bgp/inst/dom-[%s]/af-[%s]/prefix-[%s]
+ds_description: This data source can read the BGP (VRF) advertise prefix configuration.
+res_description: This resource can manage the BGP (VRF) advertise prefix configuration.
+doc_path: Routing%20and%20Forwarding/bgp:AdvPrefix/
+doc_category: BGP
+parents:
+  - BGP Address Family
```

Comment:
We should then also update the "BGP Address Family" definition to list this as a child.